src/pyj/read_book/paged_mode.pyj

```diff
@@ -435,25 +429,29 @@ def scroll_to_elem(elem):
             # elem.scrollIntoView(). However, in some cases it gives
             # inaccurate results, so we prefer the bounding client rect,
             # when possible.
-            left = elem.scrollLeft
+            # Columns start on the right side in RTL mode, so get that instead here...
+            pos = elem.scrollLeft if scroll_viewport.ltr() else elem.scrollRight
         else:
-            left = br.left
-        scroll_to_xpos(viewport_to_document(
-            left+2, elem.scrollTop, elem.ownerDocument)[0])
+            # and here.
+            pos = br.left if scroll_viewport.ltr() else br.right
+        scroll_to_xpos(scroll_viewport.viewport_to_document(
+            pos+2, elem.scrollTop, elem.ownerDocument)[0])
```

Comment:
Because of the reversal of the position internally in ScrollViewport, this will actually be +2 to the left in RTL mode and +2 to the right in LTR mode.